### ISSUE TITLE
SSL certificate expired for your personal site

### ISSUE THREAD

**Issue by rajpiskala** [Falconerd/vanilla-website#1]
SSL certificate expired on July 3, 2021, so a little over two weeks ago. Now the connection appears as not secure and gives a warning whenever the browser connects to it.

If you're hesistant to renew your SSL certificate due to the cost, I would recommend looking at Let's Encrypt as they will certify your website for free. 

![image](https://user-images.githubusercontent.com/57082175/126081064-451a1de9-9dea-4602-a486-4c0aa5ec83f2.png)

![image](https://user-images.githubusercontent.com/57082175/126081110-8223375a-9420-4b84-a7a8-4bfa460e1839.png)

**Comment by Falconerd** [comment]
Thanks for letting me know. :) Weird I didn't get any emails.

**Comment by rajpiskala** [comment]
Yeah, no problem! I'd recommend using Certbot to get it signed by "Let's Encrypt" for free  if you're hesitant on paying money for an SSL certificate.
